### PR TITLE
Translate to Japanese

### PR DESCRIPTION
**Pull request by Yamagami-Takuma**
@f-higashi  
Can you review this?

### CODE REVIEW

**Comment by cheld** [comment]
Cool :)

**Comment by bryk** [comment]
@cheld Will you finish the review and merge?

File: i18n/messages-ja.xtb [hunk 1]
```diff
@@ -39,96 +39,96 @@
...
-  <translation id="5324906937580749197" key="MSG_COMMON_NAMESPACE_NAMESPACESELECT_3" desc="Label atop a list of namespaces">Namespaces</translation>
-  <translation id="395521352310086904" key="MSG_COMMON_RESOURCE_DELETERESOURCE_0" desc="Title for a delete resource dialog.">Delete a {{::$ctrl.resourceKindName}}</translation>
-  <translation id="1507210670835642034" key="MSG_COMMON_RESOURCE_DELETERESOURCE_1" desc="Title for a delete resource dialog.">Delete a {{::$ctrl.resourceKindName}}</translation>
+  <translation id="4492076728105331074" key="MSG_COMMON_NAMESPACE_NAMESPACESELECT_0" desc="Title for namespace select.">ネームスペース</translation>
+  <translation id="5174603006239796789" key="MSG_COMMON_NAMESPACE_NAMESPACESELECT_1" desc="Text describing what namespace selector is">Selector for ネームスペース</translation>
+  <translation id="7324783266461564175" key="MSG_COMMON_NAMESPACE_NAMESPACESELECT_2" desc="Text for dropdown item that indicates that no namespace was selected.">All ネームスペース</translation>
```
Comment (Yamagami-Takuma) [inline i18n/messages-ja.xtb]: It was fixed now :)

**Comment by bryk** [comment]
@f-higashi Thanks a lot! :) 

Merging.

**Pull request merged**